ui/components/admin.js

```diff
@@ -55,11 +55,14 @@ function Grant ({ id, userID, grants }) {
 
 export default function () {
   const { data: { items: grants } = {} } = useSWR(() => '/api/grants?resource=infra&privilege=admin', { fallbackData: [] })
+  const { data: auth } = useSWR('/api/users/self')
   const { mutate } = useSWRConfig()
 
   const [name, setName] = useState('')
   const [error, setError] = useState('')
 
+  const sortedGrants = grants && [...grants?.filter((g) => g.user === auth.id), ...grants?.filter((g) => g.user !== auth.id).sort((a, b) => (a.user).localeCompare(b.user))]
```

Comment:
We should filter grants that don't have a user in this case

```suggestion
  const userGrants = grants?.filter(g => g.user)  
  const sortedGrants = [...userGrants?.filter(g => g.user === auth.id) || [], ...userGrants?.filter(g => g.user !== auth.id).sort((a, b) => a.user.localeCompare(b.user))]
```